src/components/OrgPeopleOrganizationsCard/OrgPeopleOrganizationsCard.tsx

```
@@ -0,0 +1,132 @@
+            <h6 className={styles.orgadmin}>
+              {tCommon('members')}: <span>{members.length}</span>
+            </h6>
+          </div>
+        </div>
+
+        <Button
+          onClick={handleClick}
+          data-testid="manageBtn"
+          className={styles.manageBtn}
+        >
+          {data && data?.isSampleOrganization && (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)